pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -767,6 +769,44 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 		})
 	})
 
+	It("when network VMI spec validator execute, should", func() {
```

Comment:
Could you please elaborate what this test should do?

Reply:
Fixed